sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -385,7 +383,13 @@ def _prepare_runtime_metrics(self, distributions):
     return runtime_in_s
 
 
-class ConsoleMetricsPublisher(object):
+class MetricsPublisher:
```

Comment:
what does this base class gain for us?

Reply:
work with type hint here
https://github.com/apache/beam/blob/560aa8d7b75fee87c0e7911ac19557c6f11838ae/sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py#L206
as publish method is invoked here
https://github.com/apache/beam/blob/560aa8d7b75fee87c0e7911ac19557c6f11838ae/sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py#L236